instructure/instructure-ui
# [BREAKING] remove desktopConfig prop from TopNavBar.Layout for v9

INSTUI-4016

BREAKING CHANGE:
desktopConfig prop is removed from TopNavBar.Layout

should only be merged right before the v9 release

test plan:
- check docs for TopNavBar: every mention of `desktopConfig` or `hideActionsUserSeparator` should be removed
- add `desktopConfig` prop to `<TopNavBar.Layout />`: there should be a console error/warning about the deprecated prop with a url to the upgrade guide
- check the upgrade guide
